deploy_gui.sh

```
@@ -0,0 +1,195 @@
+
+# Function to check and update environment variables
+create_or_validate_env_file() {
+  env_file=".env"
+
+  if [ ! -f "$env_file" ]; then
+    zenity --info --text="$env_file does not exist. Creating it now..."
+    touch "$env_file"
+  fi
+
+  # Source the existing .env file to retain values
+  source "$env_file"
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)